README.md

```diff
@@ -2,5 +2,21 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all Top Level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
-For more information read the [TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md).
+For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md) adopted by the Node.js Foundation Board of Directors on June 17th 2015.
 
+If your project is interested in joining the Node.js Foundation please read the [Project Lifecyle.md](./Project Lifecycle.md) documentation.
+
+## TSC Members
+
+## Top Level WG and TLPs
+
+* Working Groups
+ * Mentors
+* Top Level Projects
+ * Core TLP
```

Comment:
The “Core TLP” is pretty much the whole “project” as it stands today, so it is its own category and will have its own WGs. I’m sending a small change to make that clearer.

> On Oct 14, 2015, at 1:57PM, Ben Noordhuis <notifications@github.com> wrote:
> 
> In README.md <https://github.com/nodejs/TSC/pull/2#discussion_r42051998>:
> 
> >  
> > +If your project is interested in joining the Node.js Foundation please read the [Project Lifecyle.md](./Project Lifecycle.md) documentation.
> > +
> > +## TSC Members
> > +
> > +## Top Level WG and TLPs
> > +
> > +* Working Groups
> > + * Mentors
> > +* Top Level Projects
> > + * Core TLP
> Indent error? Or is "Core TLP" a subcategory?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/nodejs/TSC/pull/2/files#r42051998>.
>